pkg/controller/sparkapplication/controller.go

```diff
@@ -551,6 +551,21 @@ func (c *Controller) syncSparkApplication(key string) error {
 		} else {
 			appCopy = c.submitSparkApplication(appCopy)
 		}
+		// Remove just submitted application, if CRD doesn't exist anymore
+		appVerificationCopy, err := c.getSparkApplication(namespace, name)
+		if err != nil {
```

Comment:
Dnfjxmf .  I.  Jfncd k f co n co n dkcc.   Jfngkxr.  Cold no I I j. X I.  J j did be x. Dkkx d mmm mmm in fbk cj. C j. Kk co j jcn my CV for hi I'm.  Nik cc.  J